src/server/c-interop-native-extensions.md

```diff
@@ -3,535 +3,20 @@ title: Native extensions for the standalone Dart VM
 description: The original way for command-line Dart apps to call C/C++ functions.
 ---
 
-_Written by William Hesse <br>
-May 2012_
-
 {{site.alert.note}}
-The extension mechanism discussed in this page is for deep integration of the VM
-by custom Dart embedders.
-This extension mechanism is **unsupported** for extension purposes,
-and is **[being removed](https://github.com/dart-lang/sdk/issues/45451)**.
+The extension mechanism which was previously discussed on this page has been
```

Comment:
Maybe `...previously discussed on this page -- native extensions -- has been` ?

That's likely better for SEO